richtextfx/src/main/java/org/fxmisc/richtext/model/TextEditingArea.java

```diff
@@ -161,11 +198,42 @@
     void replaceText(int start, int end, String text);
 
     /**
+     * Replaces a range of characters with the given text.
+     *
+     * It must hold {@code 0 <= start <= end <= getLength()} where
+     * {@code start = getAbsolutePosition(startParagraph, startColumn);} and is <b>inclusive</b>, and
+     * {@code int end = getAbsolutePosition(endParagraph, endColumn);} and is <b>exclusive</b>.
+     *
+     * <p><b>Warning:</b> uses {@link #getAbsolutePosition(int, int)}. Be aware of possible bugs that
+     * can arise by using this method.</p>
+     *
+     * @param text The text to put in place of the deleted range.
+     * It must not be null.
+     */
+    default void replaceText(int startParagraph, int startColumn, int endParagraph, int endColumn, String text) {
+        int start = getAbsolutePosition(startParagraph, startColumn);
+        int end = getAbsolutePosition(endParagraph, endColumn);
+        replaceText(start, end, text);
+    }
+
+    /**
      * Replaces a range of characters with the given rich-text document.
      */
     void replace(int start, int end, StyledDocument<PS, S> replacement);
 
     /**
+     * Replaces a range of characters with the given rich-text document.
+     *
+     * <p><b>Warning:</b> uses {@link #getAbsolutePosition(int, int)}. Be aware of possible bugs that
+     * can arise by using this method.</p>
```

Comment:
Yeah.....
I wasn't sure how to basically say: "Hey, this method might do some quirky stuff because _you_ gave it an index that was off by one. So, keep an eye out!"

Do you have any suggestions for how to rephrase that to avoid that unwanted connotation?